src/Command/UpdateComposerPluginsCommand.php

```diff
@@ -72,7 +72,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $missedUpdates = [];
-        if (count($updates) === 0) {
+        if ($updates === []) {
```

Comment:
Note that the comparison with an empty array is the slowest method to check if the array is empty: https://stackoverflow.com/a/59841707/19497102

I could reproduce similar numbers using PHP 8.2.